docs/source/EccFramework.rst

```diff
@@ -0,0 +1,154 @@
+ECC Framework
+======================================================================================================
+
+The QECC library offers means for automatic implementation and evaluation of error-correcting codes for quantum
+computing. More precisely, the library allows to automatically apply different error correction schemes to quantum
+circuits provided as OpenQASM files or Qiskit :code:QuantumCircuit objects. The "protected" quantum circuits can then be exported again in the form of OpenQASM files or can be directly used for noise-aware quantum circuit simulation. For the latter case, we also provide a wrapper script which makes use of the provided framework to apply error correction schemes to circuits and directly simulate those circuits using qiskit.
+
+Usage
+#####
+
+Having the Python module installed, error correction can be applied to a quantum circuit using :func:~mqt.qecc.apply_ecc, like so
+
+.. code-block:: python
+  from mqt import qecc
+  from qiskit import QuantumCircuit
+
+  file = "path/to/qasm/file.qasm"  # Path to the OpenQASM file the quantum circuit shall be loaded from
+  ecc = "Q7Steane"  # Error correction code that shall be applied to the quantum circuit
+  ecc_frequency = 100  # After how many times a qubit is used, error correction is applied
+
+  result = qecc.apply_ecc(file, ecc, ecc_frequency)
+
+  circ = QuantumCircuit().from_qasm_str(result["circ"])
+  print(circ)
+
+Currently, the error correction schemes Q3Shor, Q5Laflamme, Q7Steane, Q9Shor, Q9Surface, and Q18Surface are supported.
```

Comment:
Consider making this a list for readability.

Reply:
We explicitly list all supported Eccs a few lines below. Therefore I would like to keep it as it is.